src/components/Card.js

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { PALETTE } from "../shared";
+
+import Text from "./Text";
+import SubsectionHeader from "./SubsectionHeader";
+
+const CardEl = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  border-radius: 4px;
+  border: solid 1px ${PALETTE.white80};
+  box-shadow: 0 8px 16px -8px rgba(19, 12, 51, 0.08);
+
+  ${SubsectionHeader} {
+    margin-bottom: 0.5rem;
+  }
+
+  img {
+    display: block;
+    overflow: hidden;
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    margin-bottom: 1.5rem;
+  }
+`;
+
+/**
+ * Note: This exports a React component instead of a styled-component.
+ * [Design Mockup](https://app.zeplin.io/project/5d5aecf2918cf74d46363015/screen/5d813f575d1398783f61df5b)
+ */
+
+const Card = React.forwardRef(function Card(
```

Comment:
With arrow function eslint was throwing an error `Component definition is missing display name  react/display-name`

Reply:
Ah interesting, yeah I guess it would be an anonymous function, since it's not directly assigned to `Card`